osal/src/os/freertos/inc/os-freertos.h

```diff
@@ -0,0 +1,83 @@
+/*
+ *  NASA Docket No. GSC-18,370-1, and identified as "Operating System Abstraction Layer"
+ *
+ *  Copyright (c) 2019 United States Government as represented by
+ *  the Administrator of the National Aeronautics and Space Administration.
+ *  All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * \file     os-posix.h
+ * \ingroup  posix
+ * \author   joseph.p.hickey@nasa.gov
+ *
+ * Purpose: This file contains definitions that are shared across the POSIX
+ *          OSAL implementation.  This file is private to the POSIX port and it
+ *          may contain POSIX-specific definitions.
+ */
+
+#ifndef INCLUDE_OS_POSIX_H_
```

Comment:
Change to INCLUDE_OS_FREERTOS_H_ for consistency?

Reply:
Good catch, all the comments should be changed at some point as well